integration_tests/.env

```diff
@@ -0,0 +1,2 @@
+# Set to run some specific file tests (ex. fib.cairo,alloc.cairo)
+INTEGRATION_TESTS_FILTERS=
```

Comment:
This file is more of a utility, I like the idea that I have this option but I don't think it should be part of the repo

Reply:
I have achieved that by first checking if the environment variable is set in the console, and if not, then load it from the .env file. @rodrigo-pino